services/core-web/src/components/mine/Permit/PermitConditions.tsx

```diff
@@ -184,7 +270,7 @@ const PermitConditions: FC<PermitConditionProps> = ({
           </Col>
         </Row>
       }
-    />
```

Comment:
I think this one wants to stay a self-closing tag? It doesn't take any children

Reply:
That's weird.  I made no concious decision to change that.